modules/modules.nix

```diff
@@ -99,26 +99,26 @@ let
     (loadModule ./programs/vscode/haskell.nix { })
     (loadModule ./programs/z-lua.nix { })
     (loadModule ./programs/zathura.nix { })
-    (loadModule ./programs/zsh.nix { })
+    (loadModule ./programs/zsh/zsh.nix { })
     (loadModule ./services/blueman-applet.nix { })
-    (loadModule ./services/compton.nix { })
+    (loadModule ./services/x11/compton.nix { })
     (loadModule ./services/dunst.nix { })
-    (loadModule ./services/dwm-status.nix { condition = hostPlatform.isLinux; })
-    (loadModule ./services/emacs.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./services/misc/dwm-status.nix { condition = hostPlatform.isLinux; })
```

Comment:
Seems like to make more sense to put this under the x11 directory? Could fix in NixOS as well.